test/jdk/sun/security/ssl/Stapling/java.base/sun/security/ssl/StatusResponseManagerTests.java

```diff
@@ -95,6 +102,7 @@ public static void main(String[] args) throws Exception {
         rootOcsp.stop();
     }
 
+
```

Comment:
Nit: added space

Reply:
Thanks for the review.  I just removed the space.